tests/release/test_versions.py

```
@@ -0,0 +1,75 @@
+        for name in (
+            "scripts/check-release-version.sh", "crates/mw-cli/Cargo.toml",
+            "crates/memorywhale-core/Cargo.toml", "src-tauri/Cargo.toml",
+            "src-tauri/tauri.conf.json", "package.json", "package-lock.json",
+        ):
+            destination = self.root / name
+            destination.parent.mkdir(parents=True, exist_ok=True)
+            shutil.copyfile(ROOT / name, destination)
+        self.version = json.loads((self.root / "package.json").read_text())["version"]
+
+    def run_gate(self, *args):
+        return subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)